docs/community/journey.md

```diff
@@ -46,7 +46,7 @@ The guessing game is a very basic contract that showcases some of Obscuro's capa
 
 Before we get into how the game works, let's see it in action and play. See the instructions [here](https://docs.obscu.ro/testnet/example-dapps.html)
 
-Now that we've played the game let's try deploying the Guessing game to the Obscuro testnet. You can clone the repo from [here](https://github.com/obscuronet/number-guessing-game). 
+Now that we've played the game let's try deploying the Guessing game to the Obscuro testnet. You can clone the repo from [here](https://github.com/obscuronet/sample-applications/number-guessing-game). 
```

Comment:
This link is broken. It would be https://github.com/obscuronet/sample-applications/tree/main/number-guessing-game.

Reply:
And you can't clone a subfolder like that, AFAIK. You have to clone the entire `sample-applications`.

Reply:
doh.. yes